ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -557,6 +557,36 @@ def test_tensorflow_conv3d_transpose(
         dilations=1 if not ivy.gpu_is_available() else dilation,
     )
 
+    
+@handle_cmd_line_args
+@given(
+    x_f_d_df=_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["NWC","NCW","NHWC","NCHW","NDHWC","NCDHW"]),
+        padding=st.sampled_from(["VALID", "SAME"]),
```

Comment:
Here you will need to adjust the behavior of the `_x_and_filters` helper function to make sure that the data format selected matches the type of array being generated